src/main/treeDataProviders/issuesTree/codeFileTree/eosTreeNode.ts

```diff
@@ -34,14 +36,24 @@ export class EosTreeNode extends CodeIssueTreeNode {
         return this._codeFlows;
     }
 
+    public get fullDescription(): string | undefined {
+        return this._fullDescription;
+    }
+
+    public get snippet(): string | undefined {
+        return this._snippet;
+    }
+
     public getDetailsPage(): IEosPage {
         return {
             header: this.label,
             pageType: PageType.Eos,
+            severity: SeverityUtils.toWebviewSeverity(this.severity),
             location: {
                 fileName: Utils.getLastSegment(this.parent.projectFilePath),
                 file: this.parent.projectFilePath,
-                row: this.regionWithIssue.start.line + 1,
+                snippet: this.snippet,
+                row: this.regionWithIssue.start.line,
```

Comment:
No need `this.regionWithIssue.start.line + 1`?

Reply:
no, at the constructor of the superclass we take the difference between the scanner and vscode into account.